### DISCUSSION TITLE
Do we need GitHub Issue Templates?

### DISCUSSION THREAD

**Discussion by Ryuno-Ki** [mdn/yari#3128] [Ideas]
Recently, on MDN Web Docs Matrix the user DerekNonGeneric asked for [issue templates](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) on our GitHub repo (mdn/content, in this case).

He suggested to lend on [nodejs/node issue template](https://github.com/nodejs/node/tree/3b2863db12818778671713b5c30363f788912aae/.github/ISSUE_TEMPLATE).

What's your take on it?

**Comment by chrisdavidmills** [comment]
I've actually got this on my todo list; will hopefully happen this week. I'm thinking initially just a basic bug report template, and a template for suggesting longer-form pieces of work that people would like to see on MDN. If a need is seen for other specific templates, we can add more.

**Reply by DerekNonGeneric** [reply]
I was the one who contributed the issue template for API reference docs problems that you see there. If you view [the raw source](https://raw.githubusercontent.com/nodejs/node/3b2863db12818778671713b5c30363f788912aae/.github/ISSUE_TEMPLATE/3-api-ref-docs-problem.md) (the majority of which users will end up seeing due to how issue templates work), you may find some features you like for yours.

**Reply by chrisdavidmills** [reply]
Thanks @DerekNonGeneric , I'll give it a look. Much appreciated!

**Reply by Ryuno-Ki** [reply]
Looks like issue templates are coming: https://github.com/mdn/content/pull/2906

**Comment by Ryuno-Ki** [comment]
For the record, @DerekNonGeneric is on GitHub, too :o

**Comment by schalkneethling** [comment]
We are now using issue and pull request templates and are standardizing these across our repositories. Thank you for starting the initial discussion @Ryuno-Ki.
